.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,107 @@
+XX
+name: Ubuntu Workflow
+
+on:
+  pull_request:
+    branches: [ gnucobol-3.x ]
+  push:
+    branches: [ gnucobol-3.x ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        include:
+          - os: ubuntu-latest
+            skip_test: true
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Configure git
+        run: git config --global core.symlinks false
+
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install packages
+        run: |
+          sudo apt-get install automake libtool libdb5.3-dev help2man gettext texlive
+
+      - name: Set git user
+        run: |
+          git config --global user.name github-actions
+          git config --global user.email github-actions-bot@users.noreply.github.com
+
+      - name: bootstrap
+        run: |
+          ./build_aux/bootstrap
+
+      # FIXME: With TERM="dumb" `make check` fails with:
```

Comment:
Why not setting `TERM=xterm` or ... ok, you've already set that so the FIXME seems to be outdated.